runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegistrationView/UsersRegistrationView.tsx

```diff
@@ -41,7 +41,7 @@ const UsersRegistrationView: FC = () => {
     );
 
     const { notActivated } = useSelector(usersSelector);
-    const { search, handleSearch, refreshSearch } = useUserSearch(false, limit, page);
+    const { search, handleSearch, refreshSearchNotActivated } = useUserSearch(false, limit, page);
```

Comment:
imo this file & component could be renamed to InactiveUsersList/View
similarly UsersListView/Table to ActivatedUsersList

imo with this analogy is much easier to find oneself in the code

Reply:
🙂